docs/examples/single_moving_target_relative_Doppler.ipynb

```diff
@@ -0,0 +1,216 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    radar.update_pose(Ts_s*N_s)
</pre><p>Please remove. We assume a certain initial position and constant velocity for the chirp sequence. It is not required to actually move anything.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rf-eng/scikit-radar/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>